app/adapters/application.js

```diff
@@ -1,8 +1,7 @@
 import RESTAdapter from '@ember-data/adapter/rest';
 import { inject as service } from '@ember/service';
-import AdapterFetch from 'ember-fetch/mixins/adapter-fetch';
 
-export default class ApplicationAdapter extends RESTAdapter.extend(AdapterFetch) {
+export default class ApplicationAdapter extends RESTAdapter {
```

Comment:
Ember Data has built-in support to use fetch since 3.9.2 accordingly to [ember-fetch documentation](https://github.com/ember-cli/ember-fetch/tree/v8.0.2#use-with-ember-data).